changelog.txt

```diff
@@ -0,0 +1,3 @@
+changed - `firebase` is no longer a direct dependency.
+fixed - Updated test suite to use `firebase-admin` instead of `firebase`.
```

Comment:
I would say neither of these next two things need to be in the release notes since they don't actually affect anyone who is upgrading. The test suite is irrelevant to them and the only dependencies updated were `devDependencies`.